MODULE.bazel

```diff
@@ -34,26 +34,7 @@ archive_override(
 # TODO: Remove explicit `goarch`/`goos` once rules_go v0.47.0 is released:
 # https://github.com/bazelbuild/rules_go/commit/145b042e21a073fa3637c096bf9779cdbcac6f75
 go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
-go_sdk.download(
-    goarch = "arm64",
-    goos = "linux",
-    version = "1.22.1",
-)
-go_sdk.download(
-    goarch = "amd64",
-    goos = "linux",
-    version = "1.22.1",
-)
-go_sdk.download(
-    goarch = "arm64",
-    goos = "darwin",
-    version = "1.22.1",
-)
-go_sdk.download(
-    goarch = "amd64",
-    goos = "darwin",
-    version = "1.22.1",
-)
+go_sdk.download(version = "1.22.2")
```

Comment:
@fmeum we cannot apply the same change to WORKSPACE file right?

Reply:
Yes, this change is Bzlmod only.